src/utils.php

```diff
@@ -154,3 +155,25 @@ function validateEditCarDataFields(string $name, int $year_made = 0, float $zero
     }
     return $err_msg;
 }
+
+function filterBrand(string $brand): string | false
+{
+    $model = new CarModel(make_db());
+    $cars = "";
```

Comment:
You should make this an empty array to match the datatype it will ultimately become